myko/src/lib/visualisation/sketch.js

```diff
@@ -110,6 +115,12 @@ export function draw(p5) {
     atm.edge();
   }
 
+console.log(weatherOn);
```

Comment:
If I log it it shows sometimes true sometimes false. I don't really understand why, is it inconsistent fetching or is p5.js draw function too fast?

Reply:
I tested and didn't get the problem you describe. Only thing I get is that it goes false briefly at first, then true all the way. Because the function is called twice. But it doesn't sound like that's what's happening for you @mipodi ?

Reply:
Aha I see, hmm. Well it should not become true at all because right now there are 'tillverka-aktivitet' in addedActivs, hmm